src/Form/DataTransformer/ModelToIdTransformer.php

```diff
@@ -30,33 +32,51 @@ class ModelToIdTransformer implements DataTransformerInterface
 
     /**
      * @var string
+     *
+     * @phpstan-var class-string<T>
      */
     protected $className;
 
     /**
      * @param string $className
+     *
+     * @phpstan-param class-string<T> $className
      */
     public function __construct(ModelManagerInterface $modelManager, $className)
     {
         $this->modelManager = $modelManager;
         $this->className = $className;
     }
 
-    public function reverseTransform($newId)
+    /**
+     * @param mixed|null $value
```

Comment:
Null is included in mixed.

```suggestion
     * @param mixed $value
```